From a4969f22dd472ca15e2d509717d61eaed63ae69a Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 2 Aug 2018 09:42:45 +0530
Subject: [PATCH 0391/1851] dt: bindings: media: xilinx: sdirxss : Add clock
 framework support

commit 0907e3f9489df0fb8fe2d896186fffebf90d527b from
https://github.com/Xilinx/linux-xlnx.git

The SDI Rx Subystem requires 3 clocks namely s_axi_aclk
(AXI lite interface clock), sdi_rx_clk (SMPTE SDI Rx core clock)
and video_out_clk (Video Output clock).
This patch adds the clocks and clock-names property to the
documentation.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,sdirxss.txt      | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
index 7da5c5d70bee..1e38b7d5c5a9 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,sdirxss.txt
@@ -23,6 +23,11 @@ Required properties:
 
 - xlnx,line-rate: The maximum mode supported by the design.
 
+- clocks: Input clock specifier. Refer to common clock bindings.
+
+- clock-names: List of input clocks.
+  Required elements: "s_axi_aclk", "sdi_rx_clk", "video_out_clk"
+
 - port: Video port, using the DT bindings defined in ../video-interfaces.txt.
   The SDI Rx subsystem has one port configured as output port.
 
@@ -40,6 +45,8 @@ Example:
 			xlnx,include-edh = "true";
 			xlnx,include-vid-over-axi = "true";
 			xlnx,line-rate = "12G_SDI_8DS";
+			clocks = <&clk_1>, <&si570_1>, <&clk_2>;
+			clock-names = "s_axi_aclk", "sdi_rx_clk", "video_out_clk";
 
 			ports {
 				#address-cells = <1>;
-- 
2.31.1

